modules/coo-distributed-tracing-ui-plugin-using.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-distributed-tracing-ui-plugin-using_{context}"]
+= Using the {coo-full} distributed tracing UI plugin
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+* You have installed the {coo-full}.
```

Comment:
Suppose should be 
You have installed the {coo-full} distributed tracing UI plugin